t/article_pod.t

```diff
@@ -34,7 +34,8 @@ Whatever the license is for https://github.com/perladvent/Perl-Advent
 unless( @ARGV ) {
 	# most years don't do the pod stuff the way we do it now
 	# make fancier later
-	my $y = join ',', 2011 .. 2022;
+	my $current_year = (localtime)[5] + 1900;
```

Comment:
This should work fine. Just to say that if it were me, I would use Time::Piece and avoid the 2 magic numbers. ;-)

Reply:
This was just what CoPilot suggested when I asked for code that only uses core modules. It's not pretty, though. 🤔

Reply:
Time::Piece is core, so there's no reason not to use it, IMHO.

Another reason to avoid copilot?

Reply:
<img width="513" alt="Screenshot 2024-02-16 at 11 18 58 PM" src="https://github.com/perladvent/Perl-Advent/assets/96205/6c819d6f-b6ba-49fa-af29-b7be782d7c98">

> Another reason to avoid copilot?

I've been writing TypeScript all week, testing some web forms that are subtly different. What I've been doing is writing a test, then copying that test over and making a few tweaks to the new test to do some other similar thing. Then I tell CoPilot to refactor the tests to move the common code to a different function and it "just works". It's allowing me not to have to think about the refactor, but just focus on getting the tests to work. It doesn't always get things right, but for that it works quite nicely. 

Now if I get a code review that says "you could factor out this common code", then it's just a matter of asking CoPilot to do it for me. This is my first week with CoPilot Chat, so I've been messing around and trying different things. It has taken some of the drudgery out of my daily work. That and questions that I used to have to Google and then dig around to find answers to, I'm able to get an immediate answer for that is mostly pretty good.